### DISCUSSION TITLE
大佬你README中的Edge为啥那么好看啊怎么弄的!!

### DISCUSSION THREAD

**Discussion by LonePheasantWarrior** [ChatGPTNextWeb/NextChat#2083] [General]
我的主力电脑也是Mac,我的Edge:
<img width="1667" alt="image" src="https://github.com/Yidadaa/ChatGPT-Next-Web/assets/37655011/db4ca572-1750-4412-bb33-92f285706fb9">
也想整成你那样子的

**Comment by Yidadaa** [comment]
设置里面：
- 默认皮肤
- 开启垂直标签页
